tutorial-09/MyToken.sol

```diff
@@ -35,7 +35,7 @@ contract MyFirstToken is ERC20 {
     function transferFrom(address _from, address _to, uint _value) returns (bool success) {
         if (__allowances[_from][msg.sender] > 0 &&
             _value > 0 &&
-            __allowances[_from][msg.sender] >= _value) {
+            __allowances[_from][msg.sender] >= _value && __balanceOf[_from] >= _value) {
```

Comment:
If you could put the condition on the next line I'll approve it.

Reply:
I have shifted the condition to the next line. I appreciate your concern to write clean code.

Reply:
More consistency :)